toxcore/group.c

```diff
@@ -147,12 +181,12 @@ static Group_c *get_group_c(const Group_Chats *g_c, int groupnumber)
  * TODO(irungentoo): make this more efficient.
  */
 
-static int peer_in_chat(const Group_c *chat, const uint8_t *real_pk)
+static aint peer_in_chat(const Group_c *chat, const uint8_t *real_pk)
 {
-    uint32_t i;
+    size_t i;
 
     for (i = 0; i < chat->numpeers; ++i) {
-        if (id_equal(chat->group[i].real_pk, real_pk)) {
+        if (chat->peers[i].friendcon_id != -2 && id_equal(chat->peers[i].real_pk, real_pk)) {
```

Comment:
friendcon_id < 0 means peer not connected. Also -2 means this peer almost deleted. Did you see delpeer function?
Ok, I will make a constant. But I do not see the point, because it only one special value.